control/control.go

```diff
@@ -814,7 +848,7 @@ func (p *pluginControl) FetchMetrics(ns []string, version int) ([]core.Cataloged
 }
 
 func (p *pluginControl) GetMetric(ns []string, ver int) (core.CatalogedMetric, error) {
-	return p.metricCatalog.Get(ns, ver)
+	return p.metricCatalog.GetMetric(ns, ver)
 }
 
 func (p *pluginControl) GetMetricVersions(ns []string) ([]core.CatalogedMetric, error) {
```

Comment:
I agree with @jcooklin that this one should be consider as a feature issue and maybe as `snapctl metric **gets**`. 

Currently everything is consistent, because `get` returns details on a single metric as assumed
```
snapctl metric get --help
NAME:
   **get - get details on a single metric**
USAGE:
   command get [command options] [arguments...]

OPTIONS:
   --metric-version, -v '0'     The metric version. Default (0) is latest
   --metric-namespace, -m       A metric namespace
```